build/scripts/ImportTestDataInBcContainer.ps1

```diff
@@ -40,4 +40,4 @@ foreach ($app in (Get-BcContainerAppInfo -containerName $ContainerName -tenantSp
 }
 
 # Generate demo data in the container
-Invoke-ContosoDemoTool -ContainerName $parameters.ContainerName
\ No newline at end of file
+Invoke-ContosoDemoTool -ContainerName $parameters.ContainerName -SetupData $parameters.SetupData
```

Comment:
Do you mind changing this parameter to a boolean as well? :) 
https://github.com/microsoft/BCApps/blob/3783db2c67d3dfa4f8c68727f9dbd8ebeb9529d1/build/scripts/ImportTestDataInBcContainer.ps1#L11

Alternatively, you can remove the default value.